src/user/user.controller.ts

```diff
@@ -5,58 +5,61 @@
 import * as UserErrors from '../errors/user';
 import { ServerError } from '../errors/application';
 import { IUser } from './user.interface';
-import { userModel } from './user.model';
+import { UserModel, IUserModel } from './user.model';
 import { Model } from 'mongoose';
-import { UserService } from './user.service';
 import { UserValidator } from './user.validator';
 import { EntityTypes } from '../helpers/enums';
 import { createUsers } from '../helpers/functions';
 import { Controller } from '../helpers/generic.controller';
+import UserRepository from './user.repository';
 
 export class UserController extends Controller<IUser> {
   public controllerType: EntityTypes;
   public model: Model<IUser>;
+  static _repository: UserRepository = new UserRepository();
```

Comment:
Variables should not start with underscore (unless private)
Can add to linter later